ci-operator/step-registry/hypershift/azure/create/hypershift-azure-create-chain.yaml

```diff
@@ -63,6 +70,16 @@ chain:
         ENCRYPTION_KEY_ID="$(cat "${SHARED_DIR}/azure_active_key_url")"
       fi
       
+      DISK_ENCRYPTION_SET_ID=""
+      if [[ $HYPERSHIFT_DISK_ENCRYPTION == "true" ]]; then
+        DISK_ENCRYPTION_SET_ID="$(cat "${SHARED_DIR}/azure_des_id")"
```

Comment:
if HYPERSHIFT_DISK_ENCRYPTION=false, `--disk-encryption-set-id=` looks a bit strange. Could we change to
```
EXTRA_ARGS += "--disk-encryption-set-id=${DISK_ENCRYPTION_SET_ID}"
```

Reply:
Given the existing code e.g. 
https://github.com/openshift/release/blob/6233d2dfb584364b7439e57e906ee1cd4011085f/ci-operator/step-registry/hypershift/aws/create/hypershift-aws-create-chain.yaml#L83
which has run reliably for a while and the successful rehearsals performed against this PR, I'd say this pattern is unlikely to cause any issues. However, if necessary, I can submit a follow-up PR to refactor how arguments are built.